internal/dev_server/db/sqlite.go

```diff
@@ -70,7 +72,16 @@ func (s Sqlite) UpdateProject(ctx context.Context, project model.Project) (bool,
 		return false, errors.Wrap(err, "unable to marshal flags state when updating project")
 	}
 
-	result, err := s.database.ExecContext(ctx, `
+	tx, err := s.database.BeginTx(ctx, nil)
```

Comment:
sqllite has transactions???

Reply:
Yeah. SQLite is kinda incredible. It's one of the best [tested](https://www.sqlite.org/testing.html) and [widely used](https://www.sqlite.org/mostdeployed.html) pieces of software you'll find.